### ISSUE TITLE
Error while using default settings calculate_bias

### ISSUE THREAD

**Issue by lionel68** [azizka/sampbias#5]
Thanks for your work and for putting this as a R package!

I have been trying to use it for some dataset I have my hands on, and I get the following error message:

```
b_oo <- calculate_bias(oo_df)
Adjusting to terrestrial surface...
 Done.

Creating occurrence raster...
 Done

Calculating distance raster...
Error in .local(x, y, ...) : 
  RasterLayer has no NA cells (for which to compute a distance)
In addition: Warning messages:
1: In calculate_bias(oo_df) : 'gaz' not found, using standard gazetteers
2: In dis_rast(gaz = gaz, ras = occ.out, buffer = buffer) :
  Evening buffer. Buffer set to 2
```

Here is the str of oo_df:

```
'data.frame':	1696015 obs. of  3 variables:  $ species         : Factor w/ 26 levels 
"Bachstelze","Baumpieper",..: 24 24 24 24 24 24 24 24 24 24 ...  
$ decimalLongitude: num  13.73 10.55 10.57 8.31 8.32 ...  
$ decimalLatitude : num  51.2 54 48.3 53.2 53.2 ...
--
```

**Comment by azizka** [comment]
Hi lionel,

this issue seems related to the calculation of the distance raster. Can you please check if your study area of interest (+buffer) contains at least one feature of what you want to test for (since you use the default (roads, rivers, cities, airpoirts). If this is not the case, you can:

1. provide your own, more detailed reference data via the `gaz` argument. I suspect that maybe airports might be the problem, so you can also just resample the references data to exclude airports:

```
red_gaz <- list(cities = sampbias::cities,
                      rivers = sampbias::waterbodies, 
                      roads = sampbias::roads)

calculate_bias(x, gaz = red_gaz)
```

2. Increase the buffer, so that the study area includes  at least one feature of any kind.

Please let me know if that solved the problem.

Cheers,

Alex

**Comment by lionel68** [comment]
Thanks for your swift answer. 

I created the red_gaz object as you specified, now I get some issue with the buffer:

```
b_oo <- calculate_bias(oo_df, res = 0.5, gaz = red_gaz)
Adjusting to terrestrial surface...
 Done.

Creating occurrence raster...
 Done

Calculating distance raster...
Error in dis_rast(gaz = gaz, ras = occ.out, buffer = buffer) : 
  'Buffer' is not a multiple of res. Set 'buffer' to a multiple of res
In addition: Warning message:
In dis_rast(gaz = gaz, ras = occ.out, buffer = buffer) :
  Evening buffer. Buffer set to 0.6
```

**Comment by azizka** [comment]
OK, try to set `buffer = 5`

**Comment by azizka** [comment]
This was due to internal regulations of the buffer for the distance raster calcualtion. Fixed as of version 1.0.3
